gulp/config.js

```diff
@@ -33,37 +33,29 @@ module.exports = () => {
     layouts: {
       src: `${basePaths.src}layouts/`,
     },
-    images: {
-      src: `${basePaths.content}images/`,
-      dest: `${basePaths.dest}content/images/`,
+    assets: {
+      src: [`${basePaths.src}materials/`, `${basePaths.content}`],
+      dest: `${basePaths.dest}assets`,
     },
-    logos: {
-      src: `${basePaths.assets}logos/`,
-      dest: `${basePaths.dest}assets/logos/`,
-    },
-    favicons: {
-      src: `${basePaths.assets}favicons/`,
-      dest: basePaths.dest,
-    },
-    fonts: {
-      src: `${basePaths.assets}fonts/`,
-      dest: `${basePaths.dest}assets/fonts/`,
+    meta: {
+      src: `${basePaths.src}meta/`,
+      dest: `${basePaths.dest}`,
     },
     revManifest: {
       dest: `${basePaths.dest}rev-manifest.json`,
     },
   };
 
+  const assetsFileTypes = '+(bmp|eot|flv|gif|ico|jpg|jpeg|mp4|png|svg|swf|webp|woff|woff2|xloc|xml|xpi)';
```

Comment:
hopefully not. it only exists so we don't copy definition.yml or other stuff that might hand around in the materials.